lib/cli/index.js

```diff
@@ -81,6 +80,18 @@ function cli(options) {
   debug('leek: %o', leekOptions);
 
   var leek = new Leek(leekOptions);
+
+  var cliInstance = new CLI({
```

Comment:
I did that initially, but jshint complains that we're redefining `cli` in the file, because there's already a [function named `cli`](https://github.com/ember-cli/ember-cli/blob/62e02b515809b8c566bc47c7dddeecf20171bc74/lib/cli/index.js#L36)